New PostgreSQL docs (#446)
cyclr/docs: pages/v2/05-application-connectors/information/postgresql-information.md

---
title: PostgreSQL information
sidebar: cyclr_sidebar
permalink: postgresql-information
tags: [connector-information]
linkedpage: true
---
{::options parse_block_html="true" /}

<section class="card">

## Access a table

> Cyclr uses Custom Object Categories to allow the PostgreSQL connector to access multiple database tables under a single connector installation. To access a table within a database you must create a Custom Object Category.

Reviewer: ```suggestion
Cyclr uses Custom Object Categories to allow the PostgreSQL connector to access multiple database tables with a single connector. To access a table within a database, you need to create a Custom Object Category.
```
